zadanie_domowe_MB.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+funkcja_kalkulator () {
+	echo "Podaj pierwsza wartosc liczbowa"
+	read a
```

Comment:
możesz dodać flagę -r, brak flagi to nie błąd, ale masz pewność że nic Ci się tutaj nie popsuje :D
```suggestion
	read -r a
```